### ISSUE TITLE
Want responsive Front page

### ISSUE THREAD

**Issue by SahityaRoy** [SahityaRoy/E-Learning-freesite#32]
[Website link](https://eduhub-community.github.io/Eduhub/)
i want a amazing front page for this website if anyone interested to contribute then he/she can just comment bellow then i will give the access ,

**Comment by aakankshabhende** [comment]
Do I have to make this same page responsive? @SahityaRoy

**Comment by SahityaRoy** [comment]
> Do I have to make this same page responsive? @SahityaRoy

yes make this page responsive and also if u want to add some graphics then please do so i will send u a organization link

**Comment by aakankshabhende** [comment]
I made a PR there you can check @SahityaRoy 

> > Do I have to make this same page responsive? @SahityaRoy
> 
> yes make this page responsive and also if u want to add some graphics then please do so i will send u a organization link

**Comment by MukulKolpe** [comment]
@SahityaRoy is this issue open to work?? please can I work on this?

**Comment by HIMANSHU-RA** [comment]
yes i want contribute to this front page @SahityaRoy
